### PR TITLE
Covid followup

### PR DESCRIPTION
**Pull request by davidmiller**

### CODE REVIEW

File: plugins/covid/templates/records/covid_admission.html [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  </div>
+  <div class="col-md-3">
+    [[ item.respiratory_rate ]] Breaths/min
+  </div>
+
+  <div class="col-md-3">
+    <b>Heart Rate:</b>
+  </div>
+  <div class="col-md-3">
+    [[ item.heart_rate ]] Breaths/min
```
Comment (fredkingham) [inline plugins/covid/templates/records/covid_admission.html]: Is this right?

File: plugins/covid/models.py [hunk 2]
```diff
@@ -42,3 +58,384 @@ class CovidPatient(models.Model):
...
+class CovidFollowUpCall(EpisodeSubrecord):
+    """
+    A phone call to a patient seeking follow up on their COVID-19 admission
+    """
+    _icon = 'fa fa-phone'
+
+    ONE_TO_TEN           = enum('1', '2', '3', '4', '5', '6', '7', '8', '9', '10')
+    ONE_TO_NINE          = enum('1', '2', '3', '4', '5', '6', '7', '8', '9')
+    ONE_TO_FIVE          = enum('1', '2', '3', '4', '5')
+    ZERO_TO_THREE        = (
```
Comment (fredkingham) [inline plugins/covid/models.py:359]: does enums with different fields definitely do what we think it does?

File: plugins/covid/templates/forms/itu_admission_form.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{% load forms %}
```
Comment (fredkingham) [inline plugins/covid/templates/forms/itu_admission_form.html:1]: this form was created by scaffold but we don't want it?

File: plugins/covid/models.py (hunk 2 above)
Comment (davidmiller) [reply plugins/covid/models.py:359]: ? What do you mean ? What do you think it does?

**Pull request closed without merging**